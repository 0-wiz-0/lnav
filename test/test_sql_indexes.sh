#! /bin/bash

export YES_COLOR=1

run_cap_test ${lnav_test} -n \
    -c ";EXPLAIN QUERY PLAN SELECT * FROM syslog_log WHERE log_path GLOB '*/logfile_syslog.*'" \
    ${test_dir}/logfile_syslog.*

run_cap_test ${lnav_test} -n \
    -c ";SELECT * FROM syslog_log WHERE log_path GLOB '*/logfile_syslog.*'" \
    ${test_dir}/logfile_syslog.*

