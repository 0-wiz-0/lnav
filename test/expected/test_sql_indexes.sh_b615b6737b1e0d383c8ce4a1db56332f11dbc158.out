[1m$id [0m[1m$parent [0m[1m$notused [0m[1m                 replace($detail, 'SCAN TABLE', 'SCAN')                  [0m
[1m2  [0m[1m [0m[1m0      [0m[1m [0m[1m0       [0m[1m [0m[1mSCAN all_logs VIRTUAL TABLE INDEX 1:SEARCH all_logs USING log_format = ? [0m
